### PR TITLE
Added an example for a peer network cluster setup

### PR DESCRIPTION
**Pull request by JonShelley**

### CODE REVIEW

File: examples/simple_hpc_pbs_peer_network/config.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    "install_from": "headnode",
+    "admin_user": "hpcadmin",
+    "variables": {
+        "hpc_image": "OpenLogic:CentOS-HPC:7.6:latest",
+        "location": "southcentralus",
+        "resource_group": "<NOT-SET>",
+        "vm_type": "Standard_HB60rs",
+        "compute_instances": 2,
+        "vnet_resource_group": "variables.resource_group",
+        "peer_network_resource_group": "",
```
Comment (garvct) [inline examples/simple_hpc_pbs_peer_network/config.json]: Should this be <NOT-SET>?

Comment (garvct) [inline examples/simple_hpc_pbs_peer_network/config.json]: Should this be <NOT-SET> ?

File: examples/simple_hpc_pbs_peer_network/config.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            "Network1": {
+                "resource_group": "variables.peer_network_resource_group",
+                "vnet_name": "variables.peer_vnet_name"
+            }
+        }
+    },
+    "resources": {
+        "headnode": {
+            "type": "vm",
+            "vm_type": "variables.vm_type",
```
Comment (garvct) [inline examples/simple_hpc_pbs_peer_network/config.json]: Should headnode be a cheap node like D8s_v3?

File: examples/simple_hpc_pbs_peer_network/config.json [hunk 3]
```diff
@@ -0,0 +1,118 @@
+{
+    "location": "variables.location",
+    "resource_group": "variables.resource_group",
+    "install_from": "headnode",
+    "admin_user": "hpcadmin",
+    "variables": {
+        "hpc_image": "OpenLogic:CentOS-HPC:7.6:latest",
```
Comment (garvct) [inline examples/simple_hpc_pbs_peer_network/config.json]: Probably should change this to CentOS-HPC:7.7:latest

File: examples/simple_hpc_pbs_peer_network/config.json (hunk 2 above)
Comment (JonShelley) [reply examples/simple_hpc_pbs_peer_network/config.json]: Resolved

File: examples/simple_hpc_pbs_peer_network/config.json (hunk 1 above)
Comment (JonShelley) [reply examples/simple_hpc_pbs_peer_network/config.json]: Set to <NOT-SET>

Comment (JonShelley) [reply examples/simple_hpc_pbs_peer_network/config.json]: Set to <NOT-SET>

File: examples/simple_hpc_pbs_peer_network/config.json (hunk 3 above)
Comment (JonShelley) [reply examples/simple_hpc_pbs_peer_network/config.json]: Fixed

**Pull request merged**